docs/cse/records-signals-entities-insights/signal-index-migration-faq.md

```diff
@@ -0,0 +1,257 @@
+
+## Differences compared to Signal Forwarding
+
+The new `sec_signal` index has some differences compared to the older Signal Forwarding feature. The new index:
+* Fully supports log search features such as keyword search, nested attributes, filtering by selected values, and so on.
+* Does not include the `fields` section inside the `fullRecords` object (this is because the fields are variable in length, and for some Signals will cause the log message to exceed the maximum size supported by Sumo Logic); the old Signal Forwarding included as many records that would fit (which was inconsistent).
+* Has standardized attribute names/formats (in alignment with standard Sumo Logic naming conventions as used in the `sec_record` indices). This goes beyond just the switch from “snake_case” to “camelCase”; specifically:
+     * The `asset` object has been replaced with the more accurate `entities` object. The new object is (potentially) multi-valued, and the attribute names have changed from `name` to `value`, from `asset_type` to `type`. Also, `entity_criticality` (now `criticality`) and `entity_suppressed` (now `suppressed`) have moved from the top level of the log message to the `entities` object. These values can be accessed in an expression with syntax like `| count by %entities[0].type`.
+     :::note
+     As a reminder (this is not new with the new index), `criticality` and `suppression` are representative of those values at the point in time that the Signal was created. Those values could have changed later.
+     :::
+     * Timestamps are now strings instead of integers.
+     * The `prototype` attribute is now a string instead of a boolean.
+     * The `extra` object has been deprecated. The `matchExpression` attribute contained in this object has been promoted to the top level of the log message.
+* The following attributes/objects are no longer included in the new index:
+     * `is_whitelisted` (which was deprecated in April 2022)
+     * `multiEntitySharedId` (this was an internal attribute that was not meant to be exposed to users)
+     * `new_rule_id` (duplicate of `rule_id`)
+     * The `threat_indicators` object is no longer included (this has been deprecated for some time)
+* Users may see a different number of signals for the same time period when comparing the `sec_signal` index with the old Signal Forwarding data. This is due to a number of factors related to how the Sumo platform and Cloud SIEM calculate start and end times and which timestamps are used in the different indices. But over the long term, both methods include the same signals.  
+
+The following table lists an example of the same Signal in the old format from Signal Forwarding and the new format in the `sec_signal` index:
```

Comment:
Should `table` be `code snippet `?

Reply:
Good catch, @kimsauce! I fixed it.

@pkazmir-sumo wrote the article with a table showing the code examples side-by-side. When I put his article into markdown, I placed the code examples one after the other, but forgot to change that sentence.